src/drivers/intel/cavs/ipc.c

```diff
@@ -36,6 +36,15 @@ DECLARE_SOF_UUID("ipc-task", ipc_task_uuid, 0x8fa1d42f, 0xbc6f, 0x464b,
 
 /* No private data for IPC */
 
+
+#if CONFIG_IPC_MAJOR_4
+#if CAVS_VERSION < CAVS_VERSION_1_8
+#error "Invalid configuration for IPC4."
```

Comment:
Are you sure about this? If yes, then it should be a separate patch deprecating `CAVS_VERSION_1_5` ?
In current code we only have checks:
```c
#if CAVS_VERSION == CAVS_VERSION_1_5
...
#else
...
#endif 
```

Reply:
Probably not, if this approach is ok, maybe we need to keep the 1.5 support around for IPC4.